src/index.ts

```diff
@@ -34,55 +26,51 @@ class SeqTransport extends TransportStream {
  * @param {Function} next - Continuation to respond to when complete.
  * @returns {void}
  */
-  log (info: unknown, next: () => void): void {
-    const { level, message, ...rest } =
-    <{ level:string, message: string }>info
+  log (info: any, next: () => void): void {
+    setImmediate(() => this.emit('logged', info))
+
+    // trim info to avoid passing two copies of built-in properties
+    info.exception = info.exception ? info.stack : undefined
```

Comment:
Is it valid to modify `info`, or will we upset other transports down the chain?

Reply:
I'll clone the object just in case.